Dockerfile

```
@@ -0,0 +1,44 @@
+FROM bitnami/kubectl:1.21.1 as kubectl
+FROM rancher/k3s:v1.21.1-k3s1 as k3s
+FROM mikefarah/yq:4.9.3 as yq2
+
+FROM golang:1.16.4 as base
+
+ENV GO111MODULE=on
+WORKDIR /app
+
+COPY go.mod .
+COPY go.sum .
+
+RUN go mod download
+
+COPY pkg pkg
+COPY main.go main.go
+
+FROM base AS builder
+RUN CGO_ENABLED=0 GOOS=linux GOARCH=amd64 go build -o k8s-gen .
+
+FROM golang:1.15.2-alpine3.12 as docsonnet
+
+RUN apk add --no-cache git
+RUN go get github.com/jsonnet-libs/docsonnet
+RUN go install github.com/sh0rez/docsonnet
```

TODO: After https://github.com/jsonnet-libs/docsonnet/pull/19

```suggestion
FROM jsonnet-libs/docsonnet:0.0.1 as docsonnet
```

And down below:

```diff
-COPY --from=docsonnet /go/bin/docsonnet /usr/local/bin
+COPY --from=docsonnet /usr/local/bin/docsonnet /usr/local/bin
```